presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -81,6 +87,28 @@ public void testBasicTypes()
                 .execute(getQueryRunner(), prestoCreateAsSelect("test_basic_types"));
     }
 
+    @Test
+    public void testVarbinary()
+    {
+        DataTypeTest.create()
```

Comment:
The test cases are redundant. We could move them to a method:
```java
DataTypeTest varbinaryTest = DataTypeTest.create();
addVarbinaryTests(varbinaryDataType(), "hello".getBytes(UTF_8));
varbinaryTest.execute(getQueryRunner(), prestoCreateAsSelect("test_varbinary"));

Reply:
If we add a new `apply(Consumer<DataTypeTest>)` method to `DataTypeTest`, we can keep the builder shape:
```java
DataTypeTest.create()
        .apply(TestPostgreSqlTypeMapping::addVarbinaryTests)
        .execute(getQueryRunner(), prestoCreateAsSelect("test_varbinary"));

Reply:
Done. Extracted test cases to single method.